pandas/tests/indexes/interval/test_interval.py

```diff
@@ -857,6 +858,22 @@ def test_is_all_dates(self):
         year_2017_index = pd.IntervalIndex([year_2017])
         assert not year_2017_index.is_all_dates
 
+    def test_get_value_errors(self):
+        idx = IntervalIndex.from_tuples([(1, 3), (2, 4), (3, 5), (7, 10), (3, 10)])
+        s = pd.Series(range(len(idx)), index=idx)
+
+        # key is not scalar
+        # GH 31117
+        key = [5]
+        msg = str(key)
+        with pytest.raises(InvalidIndexError, match=msg):
+            idx.get_value(s, key)
```

Comment:
`get_value` was recently deprecated in #33907, so you'll need to merge master (`git merge upstream/master`) and then ensure that we catch the `FutureWarning` associated with the deprecation:
```python
with pytest.raises(InvalidIndexError, match=msg):
    with tm.assert_produces_warning(FutureWarning):
        idx.get_value(s, key)
```

Reply:
changed